### PR TITLE
Adding api for aggregator proxy

### LINKED ISSUE #10: API for Aggregator Proxy
**Describe the bug**
We need an API which connects studios to users both at macroservice and aggregator level.

**Expected behavior**
Feign Clients should pick up the API connected to the aggregator

**Additional context**
- Make sure to have appropriate test cases(both unit and integration, also on the server) to confirm the working as well.

### PR DESCRIPTION
**Pull request by picksitquick**
Fixes #10 

## Proposed Changes

  - Added a new api to fetch user by Email Id
  - Using the updates Studio and StudioDTO Models.

### CODE REVIEW

File: src/main/java/com/gamedoora/backend/studioservices/api/StudioServicesController.java [hunk 1]
```diff
@@ -112,4 +112,11 @@ public ResponseEntity<List<StudiosDTO>> getAllStudiosByUsers(@RequestParam(requi
     public ResponseEntity<List<StudiosDTO>> getAllStudiosByUsersFirstName(@RequestParam(required = false) String user_name) {
         return createResponse(studioServicesAssembler.getAllStudiosByUsersFirstName(user_name), HttpStatus.OK);
     }
+
+    @GetMapping(
+            value = "/users/{userEmail}",
```
Comment (aprajshekhar) [inline src/main/java/com/gamedoora/backend/studioservices/api/StudioServicesController.java:117]: The URL will be /users/email as the value of the email will be coming from the request param named email. To reduce the confusion, let's name the request param as emailId

**Review by aprajshekhar — CHANGES_REQUESTED** [review]
Overall, it looks good to me.  Just one suggestion.

**Comment by picksitquick** [comment]
Sure, will update this. Also the build is failing, is it because of the token expiration?

**Comment by aprajshekhar** [comment]
@picksitquick , there are no github actions for studio services. I am not sure which repo you are referring to?

**Comment by picksitquick** [comment]
PR OUTDATED:
Will be raising a new PR as there are new changes as well.

**Pull request closed without merging**